tensorrt_llm/_torch/auto_deploy/transform/library/fused_moe.py

```diff
@@ -57,9 +76,40 @@ def _insert_fused_moe_ops(gm: GraphModule) -> int:
         node.replace_all_uses_with(new_node)
         graph.erase_node(node)
 
+        _cleanup_unstacked_weights(gm, pre_fusion_param_names)
+
     return fused_key_counter
 
 
+def _cleanup_unstacked_weights(
```

Comment:
might be great to put that into utils to be reused by other transforms like fuse_gemm